postgres-appliance/scripts/configure_spilo.py

```diff
@@ -372,6 +372,8 @@ def get_placeholders(provider):
     placeholders.setdefault('WALE_ENV_DIR', os.path.join(placeholders['PGHOME'], 'etc', 'wal-e.d', 'env'))
     placeholders.setdefault('USE_WALE', False)
     placeholders.setdefault('USE_WALG', False)
+    placeholders.setdefault('USE_WALG_BACKUP', placeholders['USE_WALG'])
```

Comment:
Briefly documenting this in the list of env vars will be of worth for other users